setup.py

```diff
@@ -12,16 +12,16 @@
     py_modules=['tap_exacttarget'],
     install_requires=[
         'funcy==1.9.1',
-        'singer-python==5.9.0',
+        'singer-python==5.12.1',
         'python-dateutil==2.6.0',
         'voluptuous==0.10.5',
         'Salesforce-FuelSDK==1.3.0'
     ],
     extras_require={
         'dev': [
             'ipdb==0.11',
-            'pylint==2.1.1',
-            'astroid==2.1.0',
+            'pylint==2.4.4',
```

Comment:
Any special requirement to upgrade pylint and astroid?

Reply:
@harshpatel4crest Please update both the modules to their latest versions currently available on pypi

Reply:
Done